packages/web_drivers/lib/firefox_driver_command.dart

```diff
@@ -0,0 +1,73 @@
+// Copyright 2020 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:args/command_runner.dart';
+import 'package:web_driver_installer/firefox_driver_installer.dart';
+
+/// Wrapper class on top of [FirefoxDriverInstaller] to use it as a command.
+class FirefoxriverCommand extends Command<bool> {
+  @override
+  String get description => 'Firefox Driver installer.';
+
+  @override
+  String get name => 'firefoxdriver';
+
+  final String defaultDriverVersion = 'fromlockfile';
+
+  FirefoxriverCommand() {
+    argParser
+      ..addFlag('always-install',
+          defaultsTo: false,
+          help: 'There might be an already installed version of the driver. '
+              'If one wants to overwrite it, set this flag')
+      ..addFlag(
+        'install-only',
```

Comment:
Thanks for the suggestion. This is the copy of the chromedriver one which is used in 3 repos now (so it will take a while if we want to migrate it). I think we should keep this the same.